apis/types/peer_create_request.go

```diff
@@ -62,51 +62,38 @@ func (m *PeerCreateRequest) Validate(formats strfmt.Registry) error {
 	return nil
 }
 
-func (m *PeerCreateRequest) validateIP(formats strfmt.Registry) error {
```

Comment:
@songshuone Please notice the comment at the top of this file.

```
 // This file was generated by the swagger tool.
 // This file was generated by the swagger tool.
 // Editing this file might prove futile when you re-run the swagger generate command
```
It seems like that we shouldn't modify it.